modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -573,43 +572,29 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 ======
 =====
 
-[[s3-seed-provider]]
```

Comment:
the same question. Is this option removed? In which Neo4j and Cypher versions?

Reply:
From Neo4j 2025.01 `s3` is supported only with the `CloudSeedProvider`. This is true for both Cypher 5 and Cypher 25 running on 2025.01. 

However, in theory it would work if a user run Cypher 5 on 2025.01 and used the `S3SeedProvider` but it is not the approach we want to encourage.